Diagnostic/RequiredExecutablesCheck.php

```diff
@@ -65,6 +65,6 @@ public function execute()
 
     private function isNohupPresent()
     {
-        return (bool) @shell_exec('which nohup');
```

Comment:
@riczorn What is the actual benefit of using `which nohup` vs `nohup --version` here ?